src/components/SendNativeToken/SendNativeTokenUI.tsx

```diff
@@ -0,0 +1,178 @@
+ * furnished to do so, subject to the following conditions:
+ * 
+ * The above copyright notice and this permission notice shall be included in all
+ * copies or substantial portions of the Software.
+ * 
+ * THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+ * IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+ * FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+ * AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+ * LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+ * OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
+ * SOFTWARE.
+ */
+import React, { useState, KeyboardEvent, useEffect } from 'react';
+
+// Services
+import {
+  EtherspotBatches,
+  EtherspotBatch,
+  EtherspotTransaction,
+  useEtherspotTransactions,
+} from "@etherspot/transaction-kit";
+
+// Utils
+import { isValidEthereumAddress } from '../../utils/validation';
+
+// Props interface for the SendNativeTokenUI component
+/**
+ * @typedef {Object} SendNativeTokenUIProps - Props for the SendNativeTokenUI component.
+ * @property {string} receiverAddress - The Ethereum address of the receiver.
+ * @property {number} chain - The Ethereum chain ID.
+ * @property {string} [className] - Additional CSS class for styling.
+ * @property {React.CSSProperties} [style] - Inline styles for the component.
+ * @property {boolean} [unstyled=false] - Flag indicating whether to apply default styles.
+ * @property {boolean} [debug=false] - Flag indicating whether to enable debugging.
+ * @property {React.ReactNode | React.RefObject<HTMLElement>} [triggerElement] - Trigger element for UI interactions.
+ * @property {boolean} [disableSendOnEnter=false] - Flag indicating whether to disable sending on Enter key press.
+ * @property {boolean} [onlyEstimate=false] - Flag indicating whether to only estimate the transaction.
+ */
+
+/**
+ * @name SendNativeTokenUI
+ * @description SendNativeTokenUI component provides a UI for sending native crypto tokens
+ * to another Ethereum address using the EtherspotTransactionKit. This user-friendly interface 
+ * is designed to simplify the complex underlying transaction mechanics, making it accessible 
+ * to users who may not be familiar with the intricacies of blockchain transactions.
+ * @param {SendNativeTokenUIProps} props - The props for the component.
+ * 
+ * @returns {React.ReactElement} The rendered component.
+ */
+
+const SendNativeTokenUI = ({
+  receiverAddress,
+  chain,
+  className,
+  style,
+  unstyled = false,
+  debug = false,
+  triggerElement,
+  disableSendOnEnter = false,
+  onlyEstimate = false,
+}: SendNativeTokenUIProps): React.ReactElement => {
+
+  // State for the amount input
+  const [amount, setAmount] = useState('0.001');
+  // State for checking if the receiver address is valid
+  const [isValidAddress, setIsValidAddress] = useState(true);
```

Comment:
This should be `false` by default, we can't assume that the default input address is correct